src/styles/index.scss

```diff
@@ -1,3 +1,124 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap';
+
+$main-accent: #060B35;
+$secondary: #616070;
+$blue-accent: #00ACDC;
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  margin: 0 auto;
+
+  width: 200px;
+  height: 408px;
+
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__imac {
+    width: 160px;
+    height: 134px;
```

Comment:
![image](https://github.com/mate-academy/layout_product-cards/assets/64101898/b578a46b-964b-4232-b5a2-e3c2aea6c43f)
You set right value but in wrong place you div and a tags give to additional redundant height